### PR TITLE
Feat/2.1.0 release changelog documentation

### LINKED ISSUE #464: v2.1.x Changelog
Before closing #458 an updated changelog between v2.0.1 and v2.1.0 should be compiled.

Release Managers:
 - @neozenith 
 - @arthurHamon2

### PR DESCRIPTION
**Pull request by arthurHamon2**
Closes #464 #465

### CODE REVIEW

**Comment by arthurHamon2** [comment]
@neozenith I let you check the changelog and the documentation changes :)

File: docs/changelog.rst [hunk 1]
```diff
@@ -1,7 +1,17 @@
 Changelog
 ---------
--  2.0.2 (UNRELEASED) - Drop support to python 3.4
+-  2.0.3 (UNRELEASED)
+-  2.0.2 - Add a `rewind` method to reset a cassette (thanks @khamidou)
```
Comment (neozenith) [inline docs/changelog.rst]: I thought because of the enhanced matcher behaviour this becomes 2.1.0 according to SemVer?

Comment (arthurHamon2) [reply docs/changelog.rst]: You're right ! Just getting the version as it was set previously.

**Comment by arthurHamon2** [comment]
@neozenith Feel free to merge and publish a new release on PyPI if everything looks good to you !

**Comment by neozenith** [comment]
5pm here... PyPI is going to be "Thursday Josh"'s problem. Thanks for your work on this.

**Comment by neozenith** [comment]
Having some issues with TravisCI timing out.... :/

**Pull request merged**